otelx/config_test.go

```diff
@@ -32,8 +32,9 @@ func TestConfigSchema(t *testing.T) {
 			Provider:    "jaeger",
 			Providers: ProvidersConfig{
 				Jaeger: JaegerConfig{
-					LocalAgentHost: "localhost",
-					LocalAgentPort: 6831,
+					LocalAgentAddress: "localhost:6831",
```

Comment:
For tests, we should check the following scenario:
if `LocalAgentAddress` is given, use it, else fallback to `LocalAgentHost` and `LocalAgentPort`. So that if all 3 are given, `LocalAgentAddress` takes precedence

Reply:
done!